sigs.yml

```diff
@@ -393,7 +393,7 @@
        - name: Morgan McLean
          github: mtwo
   - name: '.NET: SDK'
-    meeting: Every Tuesday alternating between 9:00 and 16:00 PT
+    meeting: Tuesday alternating between 09:00 PT and 16:00 PT
```

Comment:
@open-telemetry/dotnet-maintainers please confirm that both meetings are PT since the timezone was missing for the 9:00 meeting